host/sgxquote.c

```diff
@@ -12,31 +13,57 @@ OE_STATIC_ASSERT(sizeof(sgx_report_t) == 432);
 
 oe_result_t oe_sgx_qe_get_target_info(uint8_t* target_info)
 {
+    oe_result_t result = OE_OK;
     quote3_error_t err =
         sgx_qe_get_target_info((sgx_target_info_t*)target_info);
-    return (err == SGX_QL_SUCCESS) ? OE_OK : OE_PLATFORM_ERROR;
+
+    if (err != SGX_QL_SUCCESS)
+    {
+        OE_TRACE_ERROR("quote3_error_t=0x%x\n", err);
+        OE_RAISE(OE_PLATFORM_ERROR);
+    }
+
+done:
+    return result;
 }
 
 oe_result_t oe_sgx_qe_get_quote_size(size_t* quote_size)
 {
+    oe_result_t result = OE_OK;
     uint32_t* local_quote_size = (uint32_t*)quote_size;
     quote3_error_t err = sgx_qe_get_quote_size(local_quote_size);
-    return (err == SGX_QL_SUCCESS) ? OE_OK : OE_PLATFORM_ERROR;
+
+    if (err != SGX_QL_SUCCESS)
+    {
+        OE_TRACE_ERROR("quote3_error_t=0x%x\n", err);
+        OE_RAISE(OE_PLATFORM_ERROR);
+    }
+done:
+    return result;
 }
 
 oe_result_t oe_sgx_qe_get_quote(
     uint8_t* report,
     size_t quote_size,
     uint8_t* quote)
 {
+    oe_result_t result = OE_OK;
```

Comment:
The coding pattern is to initialize result to OE_FAILURE, and when everything succeeds, set it to OE_OK.

Reply:
addressed as advised.